packages/expo/ios/ExpoLiquidGlassConstantsModule.swift

```diff
@@ -0,0 +1,26 @@
+import ExpoModulesCore
+
+public class ExpoLiquidGlassConstantsModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoLiquidGlassConstants")
+
+    Constants([
+      "isLiquidGlassAvailable": ExpoLiquidGlassConstantsModule.isLiquidGlassAvailable()
+    ])
+  }
+
+  private static func isLiquidGlassAvailable() -> Bool {
+    #if compiler(>=6.2)  // Xcode 26
+    if #available(iOS 26.0, tvOS 26.0, macOS 26.0, *) {  // iOS 26
+      if let infoPlist = Bundle.main.infoDictionary {
+        // TODO(@uabx): Add a check for maximum SDK version when apple disables this flag
+        if let requiresCompatibility = infoPlist["UIDesignRequiresCompatibility"] as? Bool {
+          return !requiresCompatibility  // If the app requires compatibility then it will not use liquid glass
+        }
+      }
```

Comment:
```suggestion
    if #available(iOS 26.0, tvOS 26.0, macOS 26.0, *) {  // iOS 26
      if let infoPlist = Bundle.main.infoDictionary, let requiresCompatibility = infoPlist["UIDesignRequiresCompatibility"] as? Bool {
      // TODO(@uabx): Add a check for maximum SDK version when apple disables this flag
        return !requiresCompatibility  // If the app requires compatibility then it will not use liquid glass
    }
```

Reply:
Thanks 👍